config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Be sure to restart your server when you modify this file.
+#
+# This file contains migration options to ease your Rails 6.0 upgrade.
```

Comment:
Do we need to continue working on some of these default settings (for future Rails upgrades) or is everything done?

Reply:
You are right, we shouldn't need this file anymore.

Reply:
👍